sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AlertTriggering/TopNGroupScope.cs

```diff
@@ -6,9 +6,18 @@
 namespace Azure.AI.MetricsAdvisor.Models
 {
     /// <summary>
-    /// Alerts will only be triggered for anomalies in the top N series.
-    /// Use <see cref="Top"/> to specify the number of timestamps to take into account, and <see cref="MinimumTopCount"/>
-    /// to specify how many anomalies must be in them to send the alert.
+    /// A scope containing only the top N series group. The service determines whether a time series is part of this group
```

Comment:
It's kind of a "fixed definition" in the service, so I don't expect it to change. I tried to explain it as an algorithm because it was incredibly hard to grasp otherwise. Unfortunately, I spent hours of my life trying to understand how this works because the service lacks in documentation. These are the only things they say:

In the documentation:
> Anomalies in top N of all series: Alerts will only be triggered for anomalies in the top N series. You can set parameters to specify the number of timestamps to take into account, and how many anomalies must be in them to send the alert.

In the web portal:

![image](https://user-images.githubusercontent.com/7576220/124983135-7568a400-dfec-11eb-8bd4-ce368ceecf52.png)

Reply:
sad. but ok. better here than nowhere. Thank you for adding the description